frontend/src/libs/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback, useSearch } from '#libs/hooks/hooks.js';
+
+import { Card, Icon, Search } from '../components.js';
+import styles from './styles.module.scss';
+
+const mockedChats = [
+  { id: 1, name: 'Relationship' },
+  { id: 2, name: 'Friends' },
+  { id: 3, name: 'Family' },
+  { id: 4, name: 'Work' },
+];
+
+const mockedSelectedChat = {
+  id: 1,
+};
+
+const ChatSidebar: React.FC = () => {
+  const { filteredElements, setFilter } = useSearch(mockedChats, 'name');
+  const handleFilterChange = useCallback(
+    (search: string): void => {
+      setFilter(search);
+    },
+    [setFilter],
+  );
```

Comment:
It's useless, just pass setFilter instead of handleFilterChange

Reply:
done